packages/ensjs/src/functions/wallet/createSubname.ts

```diff
@@ -196,6 +238,26 @@ const checkCanCreateSubname = async (
     contract,
   }: Pick<BaseCreateSubnameDataParameters, 'name' | 'contract' | 'fuses'>,
 ): Promise<void> => {
+  const [resolver] = await readContract(wallet, {
+    address: getChainContractAddress({
+      client: wallet,
+      contract: 'ensUniversalResolver',
+    }),
+    abi: universalResolverFindResolverSnippet,
+    functionName: 'findResolver',
+    args: [toHex(packetToBytes(name))],
+  })
+
+  // TODO: check the interface through the resolve function
+  const isOffchain = await readContract(wallet, {
+    address: resolver,
+    abi: erc165SupportsInterfaceSnippet,
+    functionName: 'supportsInterface',
+    args: [WILDCARD_WRITING_INTERFACE_ID],
+  })
+
+  if (isOffchain) throw new OffchainSubnameError(name)
+
```

Comment:
The code will panic if it is offchain? Is that it?

Reply:
nope, it is handled [here](https://github.com/blockful-io/ensjs/pull/3/files/f32bddad697086e676cde7cc88acc6dc3cc0c005#diff-789cb9aba3ebb0260fe45a164e387afe62f26e612b17280bafe099028c03fbdaR329)